modules/test/src/test/java/io/oasp/module/test/common/base/BaseTestTest.java

```diff
@@ -25,7 +25,7 @@ protected void doSetUp() {
   @Test
   public void testInitialization() {
 
-    BaseTest test = new MyTest();
+    AbstractTest test = new MyTest();
```

Comment:
Hi @amarinso , These changes are because of renaming the class to AbstractTest.java from BaseTest. java. We can keep the name as it is .i.e, AccessControl . Pls let me know.  

Thanks,
@kiran-vadla